docs/installation-and-operation/upgrading-metabase.md

```diff
@@ -118,3 +118,23 @@ If you're on a [Metabase Cloud](https://www.metabase.com/pricing) plan, your Met
 
 - [Upgrading AWS Elastic Beanstalk deployments](running-metabase-on-elastic-beanstalk.md#deploying-new-versions-of-metabase-on-elastic-beanstalk)
 - [Upgrading Azure Web Apps deployments](running-metabase-on-azure.md#additional-configurations)
+
+## Rolling back an upgrade
+
+In general, regular backups (especially backups before upgrading), are the best policy, so we recommend reverting to a backup of your application database to roll back an upgrade. 
+
+But if you've made changes to your application database since upgrading that you want to keep, you may be able to use the `migrate down` command to roll back your Metabase application database to support the previous Metabase version you were running.
+
+To roll back your Metabase application database: 
+
+1. Change into the directory where you're running the upgraded Metabase JAR. 
+2. If you haven't already, stop the current, upgraded instance.
+3. Use the current, upgraded Metabase JAR (not the Metabase JAR you want to roll back to) to complete the rollback with the following command:
+   ```
+   java -jar metabase.jar migrate down
```

Comment:
Looks like this command also accepts an optional parameter (the target version). The default is to downgrade 1 major version.